01-python-fundamentals/01-python-fundamentals-variables.ipynb

```diff
@@ -215,9 +249,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 6,
```

Comment:
@nkorinek did you rerun the notebook from the start? i just see some errors here. in theory we want to rerun it and then clear the outputs for next year. so the rerun is just to ensure it works for you. then pust it up to the repo without the outputs populated. I just see an error below.